### PR TITLE
Overview Style (Subcomponent 2)

### PR DESCRIPTION
**Pull request by mindiweik**
- add functionality for selecting styles
- add toggle style functionality
- no tests outside of unit tests yet
- NOTE: will come back for displaying thumbnails in rows of 4 and adding a checkmark overlay for selected style

### CODE REVIEW

**Comment by mindiweik** [comment]
[Overview - Style Selector - Size: 5](https://trello.com/c/ahw732oI/18-overview-style-selector-size-5)

File: server/index.js [hunk 1]
```diff
@@ -37,18 +35,23 @@ app.get('/answers', function (req, res) {
...
       console.log('err in currentProduct get server-side', err)
     })
 })
 
+app.get('/getMetadata', (req, res) => {
+  getMetadata(req.query.productNum)
+    .then(data => res.send(data))
+    .catch(err => {
+      res.send(err)
+      console.lof('err in getMetadata server-side', err)
```
Comment (kuzmabr) [inline server/index.js:54]: Spelling error on line 54 "console.lof"

**Review by kuzmabr — APPROVED** [review]
Great job adding functionality.

**Review by thomascvan — APPROVED** [review]
looks good!

File: server/index.js (hunk 1 above)
Comment (mindiweik) [reply server/index.js:54]: Good catch! Thank you! 

Updated and committed the update.

**Comment by mindiweik** [comment]
[Overview - Style Selector Step 1 - Size: 3](https://trello.com/c/2wto9q5F/25-overview-style-selector-step-1-size-3)

**Comment by mindiweik** [comment]
[Overview - Style Selector Step 2 - Size: 3](https://trello.com/c/dzAJoDQe/26-overview-style-selector-step-2-size-3)

**Pull request merged**